packages/lab/src/radio-button/RadioButton.tsx

```diff
@@ -58,28 +43,21 @@ export const RadioButton = forwardRef<HTMLLabelElement, RadioButtonProps>(
         )}
         {...rest}
       >
-        <ControlLabel
-          {...LabelProps}
-          className={clsx(
-            withBaseName("labelContainer"),
-            {
-              [withBaseName("disabled")]: disabled,
-            },
-            className
-          )}
+        <Radio
+          checked={checked}
           disabled={disabled}
-          label={label}
-          labelPlacement="right"
+          value={value}
+          onChange={onChange}
           ref={ref}
+          id={id}
+        />
+        <Label
+          disabled={disabled}
+          className={clsx(withBaseName("label"))}
+          htmlFor={id}
```

Comment:
Why was this approach chosen over the previous approach (wrapping the control)?

Reply:
I also wondered this